test/test-RecommendationProvider.js

```diff
@@ -0,0 +1,198 @@
+/* globals require, exports */
```

Comment:
Not sure if we need these `globals`. They _should_ be covered by our ESLint `env` of `node`, I think. Or maybe I'm wrong.